src/main/update_installer.js

```diff
@@ -0,0 +1,43 @@
+
+import { dialog, shell } from "electron";
+import phin from "phin";
+const semverGreaterThan = require("semver/functions/gt");
+const { version } = require("../../package.json");
+
+const getJSON = phin.defaults({method: "GET", parse: "json", headers: {"User-Agent": "BetterDiscord Installer"}});
+
+export default async function () {
+    let downloadUrl = "https://api.github.com/repos/BetterDiscord/Installer/releases";
+    console.info(`Better Discord Installer ${version}`);
+
+    try {
+        const response = await getJSON(downloadUrl);
+        const latestRelease = response.body[0];
+        const latestVersion = latestRelease.tag_name;
+
+        if (semverGreaterThan(latestVersion, version)) {
+            console.info(`Found new release ${latestVersion}`);
+
+            const result = await dialog.showMessageBox({
+                title: "New Installer Version Available",
+                message: `A new version of the BetterDiscord installer is available. Click "Download" to download the newest version.`,
+                buttons: ["Download", "Later"],
+                defaultId: 0
+            });
+
+            if (result.response === 0) {
+                await shell.openExternal(latestRelease.html_url);
+                process.exit(0);
+            } else {
+                return false;
```

Comment:
This return value (`false`) is not used anywhere and the alternate codepath does not have a `return true` so this feels odd.